sources/Algo/src/Algo.cpp

```diff
@@ -97,6 +97,10 @@ LoadDefinitionAlgorithm::LoadDefinitionAlgorithm(Loads& loads) : NodeAlgorithm()
 
 void
 LoadDefinitionAlgorithm::operator()(const NodePtr& node) {
+  if (node->nominalVoltage < dsoVoltageLevel) {
```

Comment:
We should be careful with double comparisons, maybe you could use the doubleEquals method from dynawo?

Reply:
How could I use this doubleEquals function when I am testing a < b ?

Reply:
if (doubleNotEquals(node->nominalVoltage, dsoVoltageLevel && node->nominalVoltage < dsoVoltageLevel)

Reply:
Ok